cedar-policy/src/api.rs

```diff
@@ -783,17 +783,17 @@ impl From<cedar_policy_validator::SchemaError> for SchemaError {
             cedar_policy_validator::SchemaError::ActionEntityTypeDeclared => {
                 Self::ActionEntityTypeDeclared
             }
-            cedar_policy_validator::SchemaError::ActionEntityAttributes(e) => {
+            cedar_policy_validator::SchemaError::ActionHasAttributes(e) => {
                 Self::ActionEntityAttributes(e)
             }
             cedar_policy_validator::SchemaError::ContextOrShapeNotRecord(context_or_shape) => {
                 Self::ContextOrShapeNotRecord(context_or_shape.into())
             }
-            cedar_policy_validator::SchemaError::ActionEntityAttributeEmptySet => {
-                Self::ActionEntityAttributeEmptySet
+            cedar_policy_validator::SchemaError::ActionAttributesContainEmptySet(uid) => {
+                Self::ActionEntityAttributeEmptySet(EntityUid(uid))
             }
-            cedar_policy_validator::SchemaError::ActionEntityAttributeUnsupportedType => {
-                Self::ActionEntityAttributeUnsupportedType
+            cedar_policy_validator::SchemaError::UnsupportedActionAttributeType(uid) => {
+                Self::ActionEntityAttributeUnsupportedType(EntityUid(uid))
```

Comment:
While we're renaming the variant in Core, should we also rename the variant here?  (this would also be a breaking change)